crates/weaver_checker/src/lib.rs

```diff
@@ -81,6 +113,28 @@ impl WeaverError<Error> for Error {
     }
 }
 
+/// A list of supported policy stages.
+pub enum PolicyStage {
+    /// Policies that are evaluated before resolution.
+    BeforeResolution,
+    /// Policies that are evaluated after resolution.
+    AfterResolution,
```

Comment:
Question:  Are we planning to write a "AgainstPreviousVersion" or some such that allows us to compare two semconv resolutions against each other?

Reply:
Yes. Refining the policy engine part is my next focus since I think the documentation/code generation part is in a usable state for most use cases.

Reply:
I created a new GH issue for tracking, see https://github.com/open-telemetry/weaver/issues/168